.github/workflows/claude.yml

```
@@ -0,0 +1,64 @@
+          fetch-depth: 1
+
+      - name: Run Claude Code
+        id: claude
+        uses: anthropics/claude-code-action@beta
+        with:
+          anthropic_api_key: ${{ secrets.ANTHROPIC_API_KEY }}
+
+          # This is an optional setting that allows Claude to read CI results on PRs
+          additional_permissions: |
+            actions: read
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)